.github/workflows/poc-e2e-flow-test.yml

```diff
@@ -61,7 +62,19 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: Restore Build Cache
         uses: ./.github/actions/restore_build_cache
+      - name: Configure test tooling credentials
+        uses: ./.github/actions/setup_profile
+        with:
+          role-to-assume: ${{ secrets.E2E_TOOLING_ROLE_ARN }}
+          aws-region: us-west-2
+          profile-name: e2e-tooling
+      - name: Configure test execution credentials
+        uses: aws-actions/configure-aws-credentials@04b98b3f9e85f563fb061be8751a0352327246b0 # version 3.0.1
+        with:
+          role-to-assume: ${{ secrets.E2E_RUNNER_ROLE_ARN }}
+          aws-region: us-west-2
       - name: Run E2E tests with ${{ matrix.pkg-manager }}
         shell: bash
         run: |
           PACKAGE_MANAGER_EXECUTABLE=${{matrix.pkg-manager}} npm run test:dir packages/integration-tests/src/test-e2e/create_amplify.test.ts
+          PACKAGE_MANAGER_EXECUTABLE=${{matrix.pkg-manager}} npm run test:dir packages/integration-tests/src/test-e2e/deployment.test.ts
```

Comment:
not use `npm run e2e` at this moment because we'll add more test (e.g. `sandbox.test.ts`) later.

Reply:
I'm assuming the end goal is to replace these with a single call to `PACKAGE_MANAGER_EXECUTABLE=${{matrix.pkg-manager}} npm run e2e`?

Reply:
Yep. We'll eventually use `PACKAGE_MANAGER_EXECUTABLE=${{matrix.pkg-manager}} npm run e2e` when merge `poc/package-manager-support` to `main`

Reply:
leave the create amplify as it is and add deployment test for only npm

Reply:
:-)
I'd say "leave existing e2e tests as is, create new job with matrix to run create-amplify (with added deployment step)".